pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/regionalcomputeforwardingrule/_http.log

```diff
@@ -660,14 +660,10 @@ X-Xss-Protection: 0
   "id": "000000000000000000000",
   "kind": "compute#forwardingRule",
   "labelFingerprint": "abcdef0123A=",
-  "labels": {
-    "cnrm-test": "true",
-    "label-one": "value-one",
-    "managed-by-cnrm": "true"
-  },
```

Comment:
qq: is there follow up PR to fix this?

Reply:
We do call setLabels below...

Reply:
This is expected: 
We first create the forwarding rule without labels, and then call setLabels to add labels. So I removed labels field when the resource is created in mock server, and let setLabels to add it back-that matches the realGCP.